### ISSUE TITLE
Travis with .lintr lints packrat folder

### ISSUE THREAD

**Issue by byapparov** [r-lib/lintr#414]
Changes to `lint_package` function lead to linting of the `packrat` folder which is not really expected or needed.

**Comment by russHyde** [comment]
I can't reproduce this locally with current retl master: https://github.com/madedotcom/retl/commit/aaab3108d624d8e5dcbad2242a413f83c17c52ff

```
git clone git@github.com:madedotcom/retl.git
cd retl
Rscript -e "as.data.frame(lintr::lint_package())" | grep packrat

<-- empty -->
```

@byapparov  Could you provide some details re how you are running lintr, please. For example, are you sure you're using `lint_package`?

**Comment by byapparov** [comment]
@russHyde it is done through the Travis file. you should be able to see lint-bot messages in the conversation of referenced PR

**Comment by russHyde** [comment]
I can see the lint-bot messages in the PR, but not a call to `lint_package` in the .travis.yml.

**Comment by byapparov** [comment]
I guess, lint-bot is triggered by .lintr file

**Comment by russHyde** [comment]
Could you check if this is still an issue when running lint_package,  it should have been fixed in ...

**Comment by MichaelChirico** [comment]
Closing as this is supposed to be fixed & no update for >1 year. Please open a new issue if this is still affecting you.
